app/models/resources.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable max-len */
```

Comment:
I think we could end up having categories as another resource so wonder if the name of this should be articles as opposed to resources.  Think it would be good to keep the two separate (e.g. categories.js and articles.js) as opposed to in the same resources.js file..... thoughts ?